metadata.yaml

```diff
@@ -26,6 +26,10 @@ peers:
     interface: upgrade
 
 provides:
+  async-primary:
+    interface: async_replication
+    limit: 1
+    optional: true
```

Comment:
as followup commit: let's move to agreed `async-offer`.

Reply:
Sure. I created the following ticket for that: https://warthogs.atlassian.net/browse/DPE-4257